docs/manual-EN/2.query-language/4.statement-syntax/2.data-query-and-manipulation-statements/go-syntax.md

```diff
@@ -197,3 +198,64 @@ nebula> GO FROM 102 OVER follow BIDIRECT;
 ```
 
 The above query returns players followed by 102 and follow 102 at the same time.
+
+## Traverse From M to N Hops
+
+**Nebula Graph** supports traversing from M to N hops. When M is equal to N, `GO M TO N STEPS` is equivalent to `GO N STEPS`. The syntax is:
```

Comment:
traversing from 1 to N steps, return the edges from m to n steps.

Reply:
`GO M TO N STEPS` is equivalent to `GO N STEPS` --> ` GO N TO N STEPS` ...

Reply:
When M is equal to N, `GO M TO N STEPS` is equivalent to `GO N STEPS`.
I think it's ok here because I have assumed here M = N.

Reply:
> When M is equal to N, `GO M TO N STEPS` is equivalent to `GO N STEPS`.
> I think it's ok here because I have assumed here M = N.

OK